### ISSUE TITLE
Failure to install on Ubuntu 18

### ISSUE THREAD

**Issue by claytondukes** [userify/shim#49]
```
Creating Userify directory (/opt/userify/)
Creating uninstall script (/opt/userify/uninstall.sh)
Creating API login config (/opt/userify/creds.py)
Creating shim (/opt/userify/shim.{sh,py})
Removing exit 0 from rc.local (if there)
Checking Shim Startup

We tried to detect this Linux distro, but still unable to set start at bootup.
You'll have to set shim to startup on its own: create an init script that
launches /opt/userify/shim.sh on startup.  In most distributions, this would
have been a single line in /etc/rc.local, but you may need to do something more
exotic. Please contact us (support@userify.com) with Linux version information
so we can get working on support.


Here's some debug info, if available on your platform:

NAME="Ubuntu"
VERSION="18.04 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04 LTS
Release:	18.04
Codename:	bionic
Linux <removed> 4.15.0-20-generic #21-Ubuntu SMP Tue Apr 24 06:16:15 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by userifydevteam** [comment]
Unfortunately Ubuntu 18.04 has deprecated the rc.local file. We're working on a better fix for this now, but here is a workaround:

```
printf '#!/bin/bash\n\n\nexit 0\n' |sudo tee /etc/rc.local
sudo chmod +x /etc/rc.local
sudo systemctl enable rc-local.service
```

This should be executed before installing the shim (that is, as part of your CloudFormation/Puppet/Chef/Ansible/etc server configuration script.)

**Comment by userifydevteam** [comment]
This should be resolved by commit 251b6a6 and earlier commits and is now live on all Userify products. Please re-open if needed and thanks for using Userify!
